### ISSUE TITLE
build_site and build_reference fails when running loadPkgRdMacros

### ISSUE THREAD

**Issue by edgararuiz** [hadley/pkgdown#249]
I'm trying to build a the refernece for a package and are encountering this issue.  Tried using different packages to confirm that it wasn't a problem with the package I'm trying to document.   I'm using RStudio Pro.

Also, I think I found the fix after I forked the package.  I'm about to submit a pull request.

```
Error in tools::loadPkgRdMacros(pkg_path) : 
  argument "macros" is missing, with no default
```

**Comment by hadley** [comment]
Any chance of that PR? 😄

**Comment by hadley** [comment]
For future reference, in the PR use text like `Fixes #249` so the issue is automatically closed when the PR is merged.

**Comment by edgararuiz** [comment]
Sounds good, I will.
